File: Uri-iniciante/1012.cpp
#include<stdio.h>
/*	
*	-----------------------------------
*	|		Pedro Daniel Jardim       |
*	| 			  UFV				  | 
*	|		  25/02/2020			  |
*   ----------------------------------
*
*/



int main() 
{
		float A,B,C,pi;
		
		pi=3.14159;
		scanf("%f %f %f",&A,&B,&C);
		
		printf("TRIANGULO:%.3f \n", (A*C)/2);
		printf("CIRCULO:%.3f\n ", pi*(C*C)) ;
		printf("TRAPEZIO: %.3f\n", ((A+B)*C)/2);
		printf("QUADRADO: %.3f\n", (B*B));
		printf("RETANGULO: %.3f\n", A*B);
 

   return 0;
   
}

File: Beginner/1012.go
package main

import "fmt"

func areaTriang(a, c float64) float64 {
	return (a * c) / 2
}

func areaCircle(c float64) float64 {
	return (3.14159 * (c * c))
}
func areaTrapz(a, b, c float64) float64 {
	return ((a + b) * c) / 2
}
func areaSquad(b float64) float64 {
	return b * b
}
func areaRecta(a, b float64) float64 {
	return a * b
}

func main() {
	var a float64
	var b float64
	var c float64
	fmt.Scanf("%f %f %f \n", &a, &b, &c)
	fmt.Printf("TRIANGULO: %.3f\n", areaTriang(a, c))
	fmt.Printf("CIRCULO:  %.3f\n", areaCircle(c))
	fmt.Printf("TRAPEZIO: %.3f\n", areaTrapz(a, b, c))
	fmt.Printf("QUADRADO:  %.3f\n", areaSquad(b))
	fmt.Printf("RETANGULO:  %.3f\n", areaRecta(a, b))
}
